modules/nf-core/mcquant/main.nf

```diff
@@ -0,0 +1,36 @@
+process MCQUANT {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "labsyspharm/quantification:1.5.4"
```

Comment:
Isn't this on bioconda?

Reply:
No, not on bioconda. We might make a bioconda release in the future, but for now, this is not priority!

Reply:
Allright be sure to change the module if you were to release it :) for reproducibility sake :p